### ISSUE TITLE
Clarify when to send Unsolicited RAs 

### ISSUE THREAD

**Issue by r-pats** [furry13/slaac-renum-revive#5]
In §6.2.4 where we upgrade MAY to MUST, clarify that it's when "any information" in the PIO changes, not "configuration"

**Comment by fgont** [comment]
We mean "any configuration information". i.e., booils down as follows:
* We want to all info is sent in the minimum number of RAs.
* If *any* info changes, you want the new information to propagate quickly -- hence "MUST send... when any configuration information changes" (or the like)

**Comment by r-pats** [comment]
The issue is with the word "configuration" being ambiguous and including other unrelated device configuration.

I asked if we should be explicit with the information that would trigger it, and the feedback was to avoid being explicit, but clarify we're talking about information within the RA (even PIO?), and not all random configuration on the device.

**Comment by fgont** [comment]
Exactly: we are talkign about changes in any network configuration parameter that would make it into an RA...
